privcount/tally_server.py

```diff
@@ -143,6 +144,29 @@ def refresh_config(self):
             else:
                 ts_conf['counters'] = conf['counters']
 
+            # Load the sigma values from the data collector configuration
+            # this is a cut-down version of the DC loading script
+            dc_conf = conf['data_collector']
```

Comment:
It was easiest to send the entire config to the tally server along with the results.
(And then delete the paths of various files - unless you want to keep them?)
That way, the solution is future-proof - it doesn't matter how the sigmas are calculated, if they are stuck in the config at the end of the run, they'll make it into the context.
On both branches:
[results-add-context 98f4420] Provide all client and tally server config information as context
[add-dc-last-event eaf7644] Provide all client and tally server config information as context